Tasks/BashV3/bash.ts

```diff
@@ -128,6 +128,10 @@ async function run() {
             ignoreReturnCode: true
         };
 
+        process.on("SIGINT", () => {
+            bash.killChildProcess();
```

Comment:
Could you please add some log-message for this case?

Reply:
Thanks, I've added